changelog.txt

```diff
@@ -16,6 +16,7 @@
  * Fixed a bug in RealmQuery.isNull() and RealmQuery.isNotNull() which validated the query prematurely.
  * Fixed a bug where closed Realms were trying to refresh themselves resulting in a NullPointerException.
  * Fixed a bug that made it possible to migrate open Realms, which could cause undefined behavior when querying, reading or writing data.
+ * Fixed a bug that RealmObject accesses wrong columns in some corner cases.
```

Comment:
maybe rephrase to:
Fixed a bug causing columns indices to be wrong for some edge cases